Source/MediaInfo/Image/File_Jpeg.cpp

```diff
@@ -207,6 +207,43 @@ string Jpeg2000_Rsiz(int16u Rsiz)
     }
 }
 
+string ICC_Tag(int32u Signature)
+{
+    switch (Signature)
+    {
+        case 0x63707274: return "Copyright";
+        case 0x64657363: return "Profile description";
+        case 0x77747074: return "White point";
+        case 0x626B7074: return "Black point";
+        case 0x72545243: return "Reproduction curve, red";
+        case 0x67545243: return "Reproduction curve, green";
+        case 0x62545243: return "Reproduction curve, blue";
+        case 0x7258595A: return "Matrix, red";
+        case 0x6758595A: return "Matrix, green";
+        case 0x6258595A: return "Matrix, blue";
+        default        : return Ztring().From_CC4(Signature).To_UTF8();
+    }
+}
+
+string ICC_ColorSpace(int32u ColorSpace)
+{
+    switch (ColorSpace)
+    {
+        case 0x434D5920: return "CMY";
+        case 0x434D594B: return "CMYK";
+        case 0x47524159: return "Y";
+        case 0x484C5320: return "HLS";
+        case 0x48535620: return "HSV";
+        case 0x4C616220: return "Lab";
+        case 0x4C757620: return "Luv";
+        case 0x52474220: return "RGB";
+        case 0x58595A20: return "XYZ";
+        case 0x59436272: return "YCbCr";
+        case 0x59787920: return "xyY";

```

Comment:
Yxy?

Reply:
Fro ICC spec it is Yxy, but I see xyY at https://en.wikipedia.org/wiki/CIE_1931_color_space#CIE_xy_chromaticity_diagram_and_the_CIE_xyY_color_space
which one is the more classic?